### ISSUE TITLE
Product Licensing >> Unable to test

### ISSUE THREAD

**Issue by mnarewec** [kumulsoft/Fixed-Assets#144]
When I delete the record in MstOrganisation as advised and tried to login, it gives me runtime error.

**Comment by mnarewec** [comment]
Tried this but its not showing the activation screen. It only gives me the system administration module

**Comment by mnarewec** [comment]
This issue is resolved primarily. i will raise another issue for UI enhancement

**Comment by mnarewec** [comment]
@agilecubeteam 
Waiting on issue #293  to be resolved so I can test this issue.

**Comment by mnarewec** [comment]
This issue is resolved.
